assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2886,6 +3282,7 @@ tests:
               relation: owner
             expectation:
               - repo:1
+            errorCode: 2027 # ErrorCode_invalid_tuple
```

Comment:
Why do we have an errorCode and expectation above? The request and tuples looks fine to me.

Also, if these tests are passing, there is a false positive somehow as a result of this. We shouldn't be able to have an `expectation` and `errorCode` simultaneously and have the tests pass.

Reply:
woops

yeah the test harness should assert that if you defined `errorCode` you didn't define `expectation` and vice-versa